Running in wrapper mode for '57'

Running python job.

[name: "CPU-QUOTA"
int_val: 77
int_val: 77
int_val: 20
int_val: 63
int_val: 62
int_val: 89
int_val: 62
, name: "DISK"
int_val: 11
int_val: 12
int_val: 39
int_val: 24
int_val: 30
int_val: 18
int_val: 20
, name: "MEMORY"
int_val: 37
int_val: 10
int_val: 5
int_val: 21
int_val: 11
int_val: 27
int_val: 14
, name: "NET"
int_val: 15
int_val: 31
int_val: 33
int_val: 39
int_val: 5
int_val: 7
int_val: 19
]
The paramters are {u'CPU-QUOTA': array([77, 77, 20, 63, 62, 89, 62]), u'NET': array([15, 31, 33, 39,  5,  7, 19]), u'DISK': array([11, 12, 39, 24, 30, 18, 20]), u'MEMORY': array([37, 10,  5, 21, 11, 27, 14])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
----------------------------------------
Problem running the job:
(<class 'paramiko.ssh_exception.BadAuthenticationType'>, BadAuthenticationType('Bad authentication type', [u'publickey']), <traceback object at 0x7fd41b7c4560>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 147, in explore_spearmint
    client = re.get_client(machine)
  File "/home/ubuntu/throttlebot/src/remote_execution.py", line 6, in get_client
    client.connect(ip, username="quilt", password="")
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 424, in connect
    passphrase,
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 714, in _auth
    raise saved_exception
BadAuthenticationType: ('Bad authentication type', [u'publickey']) (allowed_types=[u'publickey'])
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 0.55 seconds.
